Blogger.Sample/Blogger.Sample.Windows/ViewModel/MainViewModel.cs

```diff
@@ -0,0 +1,34 @@
+using GalaSoft.MvvmLight;
```

Comment:
It does appear that upgrading MvvmLight added these files and references to them. I believe these are "content files" (vs. libraries) and *won't* be removed when the package is uninstalled, or overwritten by default when the package is upgraded. I think the current state of the world -- these files checked in, the reference to the libraries remains -- is correct.